doc/devices/qpu.rst

```diff
@@ -64,33 +62,18 @@ These operations include:
     pennylane_forest.ops.ISWAP
     pennylane_forest.ops.PSWAP
 
-Device options
+QVM and quilc server configuration
```

Comment:
Consider removing this part: a QPU-specific document doesn't need QVM-related stuff. (It's also duplicated in `qvm.rst`.)